test/paranoia_test.rb

```diff
@@ -1215,4 +1270,4 @@ class ParanoidBelongsTo < ActiveRecord::Base
     acts_as_paranoid
     belongs_to :paranoid_has_one
   end
-end
+end
```

Comment:
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline